radicle/src/node/tracking.rs

```diff
@@ -29,9 +29,9 @@ pub type Alias = String;
 #[derive(Default, Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Serialize, Deserialize)]
 pub enum Policy {
     /// The resource is tracked.
+    #[default]
     Track,
     /// The resource is blocked.
-    #[default]
     Block,
 }
```

Comment:
Wut

Reply:
This relates to the default `Scope` being `Trusted`. So if we take these changes in tandem then we're saying, "the default behaviour is to always track and replicate from trusted nodes". Seed nodes would then likely use `Scope::All` with `Policy::Track`. Blocking is an explicit action of saying I definitely want to block this node. Or if it's something like a personal server you would say `Scope::All` and `Policy::Block` and explicitly track your own devices or peers.

I would say that the default presented here is the most common case and each case, in turn, is less common than the previous. 

Does that make sense?